Move information into methods/cut from results re: workflow description (#178)
AlexsLemonade/ScPCA-manuscript: content/03.results.md

The workflow also performs ADT-by-cell counts matrix normalization (see Methods for details).

If a library contains ADT data, the QC report will include an additional section with a summary of ADT-related statistics, such as how many cells express each ADT, and ADT-specific diagnostic plots (Figure {@fig:figS2}B-D).
We include plots summarizing the potential effects of removing low-quality cells based on RNA and ADT counts in the QC report (Figure {@fig:figS2}B).
The first quadrant indicates which cells would be kept if the object were filtered using both RNA and ADT quality measures.
The other facets highlight which cells would be removed if filtering were done using only RNA counts, only ADT counts, or both.
The top four ADTs with the most variable expression are also identified and visualized using density plots to show the normalized ADT expression across all cells (Figure {@fig:figS2}C) and UMAPs – calculated from RNA expression data – with cells colored by ADT expression (Figure {@fig:figS2}D).

### Multiplexed libraries

To process multiplexed libraries, the HTO FASTQ files are quantified using `salmon alevin` and `alevin-fry` (Figure {@fig:figS2}C).

> The unfiltered HTO-by-cell counts matrix output from `alevin-fry` is saved as an `altExp` within the main `SingleCellExperiment`, containing the unfiltered RNA counts.

Reviewer: the comma here seemed odd, but I don't like my removal that much either.
```suggestion
The unfiltered HTO-by-cell counts matrix output from `alevin-fry` is saved as an `altExp` within the main `SingleCellExperiment` that contains the unfiltered RNA counts.
```

or maybe: 

```suggestion
As with ADT data, the HTO-by-cell counts matrix produced by `alevin-fry` is saved as an `altExp` within the main `SingleCellExperiment` object.
```

Some samples also include data from bulk RNA-seq and/or spatial transcriptomics libraries.
Both of these additional sequencing methods are supported by `scpca-nf`.
To quantify bulk RNA-seq data, `scpca-nf` takes bulk FASTQ files as input, trims reads using `fastp` [@doi:10.1093/bioinformatics/bty560], and then aligns and quantifies reads with `salmon` (Figure {@fig:figS3}A) [@doi:10.1038/nmeth.4197].

> The output is a single TSV file with the gene-by-sample counts matrix for all samples in a given ScPCA project, which is only included with project downloads on the Portal.

Reviewer: I wonder if we want to save the clause about downloads for teh section where we discuss downloads (which already includes this info)? 
```suggestion
The output is a single TSV file with the gene-by-sample counts matrix for all samples in a given ScPCA project.
```

### Merged objects

Providing data for all samples within a single file facilitates performing joint gene-level analyses, such as differential expression or gene set enrichment analyses, on multiple samples simultaneously.

> Therefore, we provide a single, merged object for each project containing all raw and normalized gene expression data and metadata for all single-cell and single-nuclei RNA-seq libraries within a given ScPCA project (with some exceptions fully described in the Methods).

Reviewer: ```suggestion
Therefore, we provide a single, merged object for each project containing all raw and normalized gene expression data and metadata for all single-cell and single-nuclei RNA-seq libraries within a given ScPCA project (with some exceptions as described in the Methods).
```

If downloading data from an ScPCA project as a single, merged file, the download will include a single `.rds` or `.h5ad` file, a summary report for the merged object, and a folder with all individual QC and cell type reports for each library found in the merged object (Figure {@fig:fig3}B).

To build the merged objects, we created an additional stand-alone workflow for merging the output from `scpca-nf`, `merge.nf` (Figure {@fig:fig3}C).
`merge.nf` takes the processed `SingleCellExperiment` objects for all single-cell and single-nuclei libraries in a given ScPCA project as input and produces a single merged gene-by-cell counts matrix containing all cells from all libraries.
No batch correction or integration is performed when creating the merged object.
Where possible, library-, cell- and gene-specific metadata found in the individual processed `SingleCellExperiment` objects are also merged.

> The merged normalized counts matrix is then used to select high-variance genes in a library-aware manner before performing dimensionality reduction with both PCA and UMAP.

Reviewer: I think moving the additional modalities section up?
```suggestion
The merged normalized counts matrix is then used to select high-variance genes in a library-aware manner before performing dimensionality reduction with both PCA and UMAP.
If additional modalities are present, these are similarly merged and included in the output object (see Methods).
```

`merge.nf` outputs the merged and processed object as a `SingleCellExperiment` object.
All merged `SingleCellExperiment` objects are converted to `AnnData` objects and exported as `.h5ad` files.

> We also account for additional modalities in `merge.nf` (see Methods).

Reviewer: If you take my previous suggestion.
```suggestion
```
